docs/api-reference/specifications/writer-object.md

```diff
@@ -0,0 +1,21 @@
+# Writer Object Specification
+
+To be compatible with `@loaders.gl/core` functions, writer objects need to conform to the following specification:
+
+## v1.0 Writer Object
+
+### Common Fields
+
+| Field           | Type        | Default    | Description |
+| ---             | ---         | ---        | ---         |
+| `name`          | `String`    | Required   | Short name of the loader ('OBJ', 'PLY' etc) |
+| `extension`     | `String`    | Required   | Three letter (typically) extension used by files of this format |
+| `category`      | `String`    | Optional   | Indicates the type/shape of data |
+
+
+### Test Function
```

Comment:
- `Encoder Function`?
- Side note: the current DracoEncoder does not directly accept the returned object from the parser. We should probably clarify here what the input to the encoder functions are expected to be.

Reply:
Thanks, added a note that the format is loader/category specific.

> The current DracoEncoder does not directly accept the returned object from the parser

Yes good point! Maybe we can fix that, if not we need to call it out in the Draco docs. To my mind, the key is to make sure we offer `DracoLoader/DracoWriter` objects that are compatible with each other (and the rest of the loaders.gl system) (This treating Draco as just another mesh/point cloud format rather than giving it special status as a compressor).

The existing Draco classes I see more as internal helpers or classes for apps that need more control.